library/runtime/src/commonMain/kotlin/io/matthewnelson/kmp/tor/runtime/internal/process/TorDaemon.kt

```diff
@@ -187,6 +187,21 @@ internal class TorDaemon private constructor(
             Process.Builder(command = paths.tor.path)
                 .args(cmdLine)
                 .environment { putAll(generator.environment.processEnv) }
+                .onError { t ->
+                    if (t.cause is UncaughtException) {
+                        // OutputFeed was passed to event observers
+                        // and one threw exception. That exception
+                        // was caught by the UncaughtException.Handler
+                        // and piped to RuntimeEvent.ERROR observer. If
+                        // the ERROR observer threw (or one was not
+                        // subscribed), the UncaughtException was thrown
+                        // within the OutputFeed.onOutput lambda and piped
+                        // here. Throw it and shut things down.
+                        throw t.cause
+                    } else {
+                        NOTIFIER.e(t)
```

Comment:
Wrap in try/catch and if `ProcessException.context` is `destroy`, do not throw if the observer throws.

Reply:
Resolved in commit 4abbe07